recipes/sepp/meta.yaml

```diff
@@ -1,11 +1,11 @@
-{% set version = "4.5.1" %}
-{% set sha256 = "51e052569ae89f586a1a94c804f09fe1b7910a3ffff7664e2005f18c7d3f717b" %}
+{% set version = "4.5.4" %}
+{% set sha256 = "e30cfffd13adb904ef4d1d6fef3d72a49f5269438c8bc4f6fd0924abaa5d75eb" %}
 package:
   name: sepp
   version: '{{version}}'
 
 source:
-  - url: https://github.com/smirarab/sepp/archive/{{version}}.tar.gz
+  - url: https://github.com/smirarab/sepp/archive/v{{version}}.tar.gz
```

Comment:
PR description is outdated I think, since this new version was released. Can you please update the description for our future reference since there is a lot going on in different PRs and issues in several repos?

Reply:
done

Reply:
Excellent, thanks!